app/utils/steam/steamcmd/wrapper.py

```diff
@@ -28,7 +28,7 @@ class SteamcmdInterface:
     Create SteamcmdInterface object to provide an interface for SteamCMD functionality
     """
 
-    _instance: Optional["SteamcmdInterface"] = None
+    _instance = None  # type: SteamcmdInterface
 
```

Comment:
Pylance doesn't like this.

To fix it, I'd do. Pylance and MyPy seem to be okay with it.

```suggestion
    _instance: "None | SteamcmdInterface" = None

```